lib/WUI/wui.c

```diff
@@ -153,9 +158,38 @@ struct altcp_pcb *prusa_alloc(void *arg, uint8_t ip_type) {
             ip_set_option((struct tcp_pcb *)result->state, SOF_REUSEADDR);
         }
         return result;
-    } else if (active_device_id == NETDEV_ESP_ID) {
-        return altcp_esp_new_ip_type(ip_type);
     } else {
         return NULL;
     }
+
+    /*
+    // Test throughtput with TCP connection
+    _dbg("Create TCP socket");
+    int sockfd = lwip_socket(AF_INET, SOCK_STREAM, 0);
+    _dbg("Sock FD: %d", sockfd);
+    struct sockaddr_in servaddr;
+    memset(&servaddr, 0, sizeof(struct sockaddr_in));
+    servaddr.sin_family = AF_INET;
+    servaddr.sin_addr.s_addr = inet_addr("10.42.0.1");
+    servaddr.sin_port = htons(5005);
+
+    if (lwip_connect(sockfd, (struct sockaddr*)&servaddr, sizeof(servaddr)) != 0) {
+        _dbg("TCP connection failed...");
+    }
+
+
+    _dbg("Sending 1MB to the TCP connection");
+    #define PACKET_SIZE (256)
+    char *buff[PACKET_SIZE];
+    for(uint i = 0; i < 4096; i++) {
+        _dbg("Sending packet %d", i);
+        lwip_write(sockfd, buff, PACKET_SIZE);
+    }
+    lwip_close(sockfd);
+
+    for(uint i = 0;; i++) {
+        osDelay(1000);
+    }
+
+*/
```

Comment:
Leftover?

Reply:
I thought it was there before. Actually, it turned out it is my code. Removed.